lib/utils/auth_exception_handler.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:dio/dio.dart';
+
+class AuthExceptionHandler {
```

Comment:
great work .. but all these messages can't send to the user so where we can use these messages.
search more how to handle exceptionHandling with Bloc

Reply:
??

Reply:
done